app/src/main/java/com/nadosunbae_android/presentation/ui/sign/SignInActivity.kt

```diff
@@ -152,9 +147,9 @@ class SignInActivity : BaseActivity<ActivitySignInBinding>(R.layout.activity_sig
             signUpBasicInfoViewModel.signIn.observe(this) { its ->
 
```

Comment:
코틀린에서 익명함수의 경우 매개변수가 한개라면 변수명을 생략가능합니다(it으로 참조 가능)! 따라서
```suggestion
            signUpBasicInfoViewModel.signIn.observe(this) { 

```
이렇게 코드를 변경하게 되면 **it**으로 해당 변수에 접근할 수 있습니다. 변수명을 명시해줄때는 의미있게 지정하는게 좋겠죠! 만약 별다른 의미가 없다면 생략해도 괜찮을 것 같습니다!

Reply:
익명함수에 대해 제대로 이해하지 못했는데 덕분에 감이 잡힌 것 같습니다. 좋은 코드 리뷰 감사합니다! 코드리뷰 반영 후 조금 더 공부해보겠습니다 :)